package.json

```diff
@@ -13,9 +13,11 @@
     "@tanstack/react-query-devtools": "^5.0.0-alpha.91",
     "axios": "^1.7.2",
     "cors": "^2.8.5",
+    "formik": "^2.4.6",
     "next": "14.2.4",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "yup": "^1.4.0"
```

Comment:
We should remove the "^" because we need the libraries to be stable and not have surprises with breaking changes. let's remove it from all the libraries as well

Reply:
solved at [6c63366](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/6c63366ce437096d199fa5b5aafd3880fbd91fc9)